bouncer/tests/broker_level_screening.ts

```diff
@@ -427,119 +540,112 @@ async function setWhitelistedBroker(brokerAddress: Uint8Array) {
-      // the transaction is refunded because the extrinsic is submitted too late.
-      async (txId) => {
-        await sleep(MILLI_SECS_PER_BLOCK * 2);
-        await markTxForRejection(txId, 'Bitcoin');
-      },
-    );
+  // -- Test vault swap rejection --
+  logger.info('Testing broker level screening for Bitcoin vault swap...');
+  const btcRefundAddress = await newAssetAddress('Btc');
 
-    await observeEvent(logger, 'bitcoinIngressEgress:DepositFinalised', {
-      test: (event) => event.data.channelId === channelId,
-    }).event;
+  await brokerLevelScreeningTestBtcVaultSwap(
+    testContext,
+    '0.2',
+    false,
+    btcRefundAddress,
+    async (txId) => setTxRiskScore(txId, 9.0),
+  );
 
-    logger.debug(`Bitcoin swap was executed and transaction was not refunded 👍.`);
+  // Currently this event cannot be decoded correctly, so we don't wait for it,
+  // just wait for the funds to arrive at the refund address
+  // await observeEvent('bitcoinIngressEgress:TransactionRejectedByBroker').event;
+  if (!(await observeBtcAddressBalanceChange(btcRefundAddress))) {
+    throw new Error(`Didn't receive funds refund to address ${btcRefundAddress} within timeout!`);
   }
 
-  // 4. -- Restore mockmode --
-  await setMockmode(previousMockmode);
+  logger.info(`Bitcoin vault swap was rejected and refunded 👍.`);
 }
 
-export async function testBrokerLevelScreening(testContext: TestContext) {
+export async function testBrokerLevelScreening(
+  testContext: TestContext,
+  testBoostedDeposits: boolean = false,
+) {
   await ensureHealth();
   const previousMockmode = (await setMockmode('Manual')).previous;
 
+  // NOTE: we don't test FLIP rejections because they are currently disabled in the
+  // deposit monitor, since Elliptic doesn't provide Flip analysis.
+
   // test rejection of swaps by the responsible broker
-  await Promise.all([
-    testBrokerLevelScreeningBitcoin(testContext),
-    testBrokerLevelScreeningEthereum(testContext, 'Eth', async (txId) =>
-      setTxRiskScore('Ethereum', txId, 9.0),
-    ),
-    testBrokerLevelScreeningEthereum(testContext, 'Usdt', async (txId) =>
-      setTxRiskScore('Ethereum', txId, 9.0),
-    ),
-    // Re-enable when DM supports FLIP
-    // testBrokerLevelScreeningEthereum(testContext, 'Flip', async (txId) =>
-    //   setTxRiskScore('Ethereum', txId, 9.0),
-    // ),
-    testBrokerLevelScreeningEthereum(testContext, 'Usdc', async (txId) =>
-      setTxRiskScore('Ethereum', txId, 9.0),
-    ),
-  ]);
+  await Promise.all(
+    [
+      testEvm(testContext, 'Eth', async (txId) => setTxRiskScore(txId, 9.0)),
+      testEvm(testContext, 'Usdt', async (txId) => setTxRiskScore(txId, 9.0)),
+      testEvm(testContext, 'Usdc', async (txId) => setTxRiskScore(txId, 9.0)),
+      testEvm(testContext, 'ArbUsdc', async (txId) => setTxRiskScore(txId, 9.0)),
+      // testEvm(testContext, 'ArbEth', async (txId) => setTxRiskScore(txId, 9.0)),
```

Comment:
Is this (and the other ArbEth test) commented out on purpose?

Reply:
Sorry, late response... 

I think that ArbUsdc probably covers the functionality already, so it's probably ok to leave it commented out for now. Maybe just add a mention that it's commented deliberately (and log an issue to track this).